prov/efa/src/efa.h

```diff
@@ -458,9 +456,11 @@ void efa_cq_inc_ref_cnt(struct efa_cq *cq, uint8_t sub_cq_idx);
 /* Caller must hold cq->inner_lock. */
 void efa_cq_dec_ref_cnt(struct efa_cq *cq, uint8_t sub_cq_idx);
 
-uint32_t *efa_av_lookup_latest_connid(struct efa_av *av, uint16_t ahn, uint16_t qpn);
+struct rxr_pkt_entry;
+
+fi_addr_t efa_av_reverse_lookup_rdm(struct efa_av *av, uint16_t ahn, uint16_t qpn, struct rxr_pkt_entry *pkt_entry);
```

Comment:
because this function is for rdm, so I think passing a `rxr_pkt_entry` make more sense. Also, I plan to eventually move the code that handle `pkt_entry->addr` in `rxr_pkt_handl_recv_completion` into this funciton, which requires `pkt_entry`.

Reply:
Actually it is not needed. I removed it in new revision.